public/JS/lobby.js

```diff
@@ -0,0 +1,31 @@
+ $(function () {
+/*--------------------------------------------------- Click events ---------------------------------------------------*/
+
+  //Button that directs user from lobby to home screen
+  $('#lobbyToHomeButton').click(function(){
+    $('#lobby').hide();
+    $('#home').show();
+  })
+
+  //Button that directs user from lobby to chat creation screen and tells server to retrieve a list of users
+  $('#createChatButton').click(function(){
+    $('#lobby').hide();
+    $('#chatCreate').show();
+    socket.emit('get users for create chat', {});
+  })
+
+  //TODO: Why is there two of these click functions?
```

Comment:
This function should be removed as the above one has the correct socket event included.

Reply:
Oh okay. I removed this function